app/components/loginlogo/loginlogo.js

```diff
@@ -14,16 +14,22 @@
  * not, you can obtain one from Tidepool Project at tidepool.org.
  */
 
+
 var React = require('react');
 
 var logoSrc = require('./images/tidepool-logo-880x96.png');
+var logoSrcDiabeloop = require('./images/diabeloop/logo_diabeloop_bleu.png');
```

Comment:
This can do the job : 
```
if(__BRANDING__ !== 'tidepool'){
  logoSrc = require('./images/'+__BRANDING__+'/logo.png');
}
```
And __BRANDING__ can also be the alternate text

Reply:
As the application name may have a different name, I would prefer a variable that contains the alternate text. It gives us more flexibility.

Reply:
I added a constant for display alt text and links